examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+"""
+Mask and Plot Remote Sensing Data with EarthPy
+==============================================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: Below we walk through a typical workflow using Landsat data with EarthPy.
+
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial, plot and mask.
+
+from glob import glob
+import os
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+import earthpy.mask as em
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+###############################################################################
+# Import Example Data
+# ------------------------------
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band) and import the ``landsat_qa`` layer which provides
+# the locations of cloudy and shadowed pixels in the scene.
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the landsat bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_paths_pre = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/*band*.tif"
+)
+landsat_paths_pre.sort()
+arr_st, meta = es.stack(landsat_paths_pre)
+
+# Import the landsat qa layer
+with rio.open(
+    "data/cold-springs-fire/landsat_collect/LC080340322016070701T1-SC20180214145604/crop/LC08_L1TP_034032_20160707_20170221_01_T1_pixel_qa_crop.tif"
+) as landsat_pre_cl:
+    landsat_qa = landsat_pre_cl.read(1)
+    landsat_ext = plotting_extent(landsat_pre_cl)
+
+###############################################################################
+# Plot Histogram of Each Band in Your Data
+# ----------------------------------------
+
```

Comment:
You'll need to delete this blank space to render the text below outside of the python code chunk.
```suggestion
```